bundles/org.openhab.automation.pythonscripting/README.md

```diff
@@ -578,12 +575,12 @@ The folder "conf/automation/python/" must be writeable by openHAB.
 
 #### Failed to inject import wrapper
 
-The reading the Python source file "conf/automation/python/lib/openhab/__wrapper__.py" failed.
+The reading the Python source file "conf/automation/python/lib/openhab/\<wrapper>.py" failed.
```

Comment:
The file is called `__wrapper__.py`:

https://github.com/openhab/openhab-python/blob/main/src/__wrapper__.py

Reply:
```suggestion
The reading the Python source file "conf/automation/python/lib/openhab/\__wrapper\__py" failed.
```

Reply:
Maybe just use backticks?

Escaping seems to depend on the Markdown renderer implementation:

https://babelmark.github.io/?text=The+reading+the+Python+source+file+%22conf%2Fautomation%2Fpython%2Flib%2Fopenhab%2F%5C__wrapper%5C__.py%22+failed.

Reply:
This is already fixed in my upcoming pythonscripting next version

https://github.com/HolgerHees/openhab-addons/blob/52fb5a27ca80dbeb87b2dd37be96c9b747d2e4a2/bundles/org.openhab.automation.pythonscripting/README.md?plain=1#L139